Watchman.Discord.UnitTests/Users/UsersControllerTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using AutoFixture.NUnit3;
+using Devscord.DiscordFramework.Commons;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+using Devscord.DiscordFramework.Services;
+using Devscord.DiscordFramework.Services.Factories;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Watchman.Discord.Areas.Users.BotCommands;
+using Watchman.Discord.UnitTests.TestObjectFactories;
+
+namespace Watchman.Discord.UnitTests.Users
+{
+    internal class UsersControllerTests
+    {
+        private readonly TestControllersFactory testControllersFactory = new();
+        private readonly TestContextsFactory testContextsFactory = new();
+
+        [Test, AutoData]
+        public async Task GetAvatar_ShouldDisplayUserAvatar(AvatarCommand command)
```

Comment:
To jest test dla normalnego przypadku użycia. Jeszcze musisz przetestować przypadek gdzie użytkownik nie ma avatara.

Reply:
Jeszcze przydałby sie przypadek gdzie sprawdzamy avatara innego użytkownika